cmd/convox-env/main.go

```diff
@@ -39,27 +35,16 @@ func main() {
 	// make it slightly harder to fetch these creds later
 	os.Unsetenv("AWS_CONTAINER_CREDENTIALS_RELATIVE_URI")
 
-	cmd.Env = append(os.Environ(), cenv...)
-
-	cmd.Stdin = os.Stdin
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
+	env := append(os.Environ(), cenv...)
 
-	err = cmd.Run()
+	err = exec(os.Args[1], os.Args[2:], env)
 
-	switch t := err.(type) {
-	case *exec.ExitError:
-		if ws, ok := t.Sys().(syscall.WaitStatus); ok {
-			os.Exit(ws.ExitStatus())
-		} else {
-			os.Exit(1)
-		}
-	case nil:
-		os.Exit(0)
-	default:
+	if err != nil {
 		fmt.Fprintf(os.Stderr, "ERROR: %s\n", err)
 		os.Exit(1)
 	}
+
+	os.Exit(0)
```

Comment:
is this exit necessary?

Reply:
nope, it should just exit cleanly if it gets to this point. good call, will remove